llama_stack/apis/datatypes.py

```diff
@@ -9,6 +9,20 @@
 from llama_models.schema_utils import json_schema_type
 
 
+@json_schema_type
+class RegisteredApi(Enum):
```

Comment:
Ah this is not the correct fix. You should not introduce this new enum, but look at  [builtin_automatically_routed_apis](https://github.com/meta-llama/llama-stack/blob/main/llama_stack/distribution/distribution.py#L24) to filter out things.

Reply:
Great suggestion. I just updated to reuse `builtin_automatically_routed_apis`. Just tested the change as well.